packages/compute-baseline/src/baseline/index.test.ts

```diff
@@ -33,6 +34,25 @@ describe("computeBaseline", function () {
     chaiJestSnapshot.configureUsingMochaContext(this);
   });
 
+  // XXX: place this somewhere useful
```

Comment:
A TODO for this is fine, but can you change "hmm" to something more final as part of this review?

Reply:
These are duplicates/predecessors of the tests. I've removed them with 5da6c91cb88bead0f45acdc4bf6efc6631caab7e.